non-oss/pharos_pro/license_key.rb

```diff
@@ -0,0 +1,93 @@
+# frozen_string_literal: true
+
+require 'time'
+require 'jwt'
+require 'ostruct'
+
+module Pharos
+  class LicenseKey
+    def self.jwt_public_key
```

Comment:
I don't think we need this on cli side at all. CLI does not need to verify the license, it can parse the details just vie base64 decoding.